### DISCUSSION TITLE
Layout reset after drag

### DISCUSSION THREAD

**Discussion by royassis** [reaviz/reagraph#209] [Q&A]
Hey

Is it possible to reset the layout after drag ?

**Comment by amcdnl** [comment]
I'm not sure if its possible right now ( or at least easy to do ). We should add this - happy to accept a PR that would do a forced relayout via a ref method.

**Comment by svenpieper** [comment]
@royassis @amcdnl Is that still not possible? I've tried a few things and unfortunately haven't found a proper solution except to temporarily set the layout to “None” and then back to the actual layout. That works, but of course I get an error in the browser log. However, it doesn't work when I cluster nodes. Clustering requires the forceddirection layout, which is why I use the “none” trick.
